tests/baselines/reference/shebangError.errors.txt

```diff
@@ -1,14 +1,17 @@
-tests/cases/compiler/shebangError.ts(2,1): error TS1127: Invalid character.
+tests/cases/compiler/shebangError.ts(2,1): error TS1128: Declaration or statement expected.
+tests/cases/compiler/shebangError.ts(2,2): error TS1127: Invalid character.
 tests/cases/compiler/shebangError.ts(2,2): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
 tests/cases/compiler/shebangError.ts(2,12): error TS2304: Cannot find name 'env'.
 tests/cases/compiler/shebangError.ts(2,16): error TS1005: ';' expected.
 tests/cases/compiler/shebangError.ts(2,16): error TS2304: Cannot find name 'node'.
 
 
-==== tests/cases/compiler/shebangError.ts (5 errors) ====
+==== tests/cases/compiler/shebangError.ts (6 errors) ====
     var foo = 'Shebang is only allowed on the first line';
     #!/usr/bin/env node
-    
+    ~

```

Comment:
shouldn't give two errors. not sure which one is better though. If it's easy to detect, adding a shebang-specific one would  be nice.

Reply:
Nice! It was pretty easy to give a clearer error message here:

> `!!! error TS18026: A hashbang ('!#') can only be used at the start of a file.`

Calling it a 'hashbang' to align with https://github.com/tc39/proposal-hashbang.

Can I mark as resolved?

Reply:
TODO: create a bug for this